private/ct.bzl

```diff
@@ -118,7 +118,9 @@ def _impl(ctx):
         log_dir = ct_logdir if ct_logdir != "" else "${TEST_UNDECLARED_OUTPUTS_DIR}"
 
         output = ctx.actions.declare_file(ctx.label.name)
-        script = """set -eo pipefail
+        script = """#!/bin/bash
```

Comment:
I believe `/usr/bin/env bash` is more broadly compatible with different unix OSes (*BSDs if I remember correctly, but also NixOS)